presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -308,4 +329,15 @@ protected T computeNext()
             }
         };
     }
+    // Copied from AbstractTestOrcReader:
+    private static List<Double> doubleSequence(double start, double step, int items)
+    {
+        List<Double> values = new ArrayList<>();
```

Comment:
nit: this can be shortened to 

```
        return IntStream.range(0, items)
                .mapToDouble(i -> start + i * step)
                .boxed()
                .collect(toImmutableList());
```